src/python_testing/TC_CADMIN_1_4_noreset.py

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/env -S python3 -B
```

Comment:
Thank you, 
I was not aware that I could do that in this manner, so I have changed the CI arguments as you have mentioned and removed the prior TC_CADMIN_1_4_noreset test module as it will no longer be needed.